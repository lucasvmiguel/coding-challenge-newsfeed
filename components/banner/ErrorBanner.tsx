import Banner from './Banner'

type Props = {
  message: string
}

export default function ErrorBanner({ message }: Props) {
  return (
    <Banner color="white" bgColor="#bc1a1b" data-test="error-banner">
      {message}
    </Banner>
  )
}
