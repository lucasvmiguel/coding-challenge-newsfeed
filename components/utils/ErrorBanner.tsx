import Banner from './Banner'

type Props = {
  message: string;
}

export default function ErrorBanner({ message }: Props) {
  return (
    <Banner bgColor="#bc1a1a" color="white">
      {message}
    </Banner>
  )
}
