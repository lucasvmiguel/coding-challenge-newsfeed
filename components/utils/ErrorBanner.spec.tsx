import renderer from 'react-test-renderer'

import ErrorBanner from './ErrorBanner'

it('renders ErrorBanner component correctly - snapshot', () => {
  const tree = renderer
    .create(<ErrorBanner message="testing" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
