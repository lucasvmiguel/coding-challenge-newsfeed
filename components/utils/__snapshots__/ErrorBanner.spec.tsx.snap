// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ErrorBanner component correctly - snapshot 1`] = `
<div
  className="sc-bdfBQB bfHpLq"
  color="white"
  data-test="error-banner"
>
  testing
</div>
`;
